libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -888,11 +888,8 @@ bool GCS_MAVLINK::send_gps_raw(AP_GPS &gps)
         return false;
     }
 
-    if (gps.highest_supported_status(0) > AP_GPS::GPS_OK_FIX_3D) {
-        if (HAVE_PAYLOAD_SPACE(chan, GPS_RTK)) {
-            gps.send_mavlink_gps_rtk(chan);
-        }
-
+    if (HAVE_PAYLOAD_SPACE(chan, GPS_RTK)) {
```

Comment:
Only send this if there is a GPS instance to minimize telemetry consumption for someone operating without an RTK capable GPS.

I'm also on the fence about the removal of the highest_supported_status() check, as it makes no sense to send the message to a GCS if the GPS can't support a RTK fix. I realize that at this point we almost have more instances that can then can't (particularly since u-blox is now capable of it), but it would be good to minimize messages that aren't applicable.

In fact my biggest hesitation on this whole PR is that it increases the link budget a bit for the most common use case. However short of adding a helper that says ```has_mavlink_rtk_payload()``` I don't see a huge advantage. (And the helper would be messy, so I guess I'm just pointing it out).